packages/components/Badge/src/CounterBadge/CounterBadge.tsx

```diff
@@ -13,7 +13,20 @@ import { stylingSettings } from './CounterBadge.styling';
 import type { CounterBadgeType, CounterBadgeProps } from './CounterBadge.types';
 import { counterBadgeName } from './CounterBadge.types';
 import { useCounterBadge } from './useCounterBadge';
-import { badgeLookup } from '../Badge';
+
+export const counterBadgeLookup = (layer: string, userProps: CounterBadgeProps): boolean => {
+  return (
+    userProps[layer] ||
+    layer === userProps['appearance'] ||
+    layer === userProps['size'] ||
+    (!userProps['size'] && layer === 'large') ||
```

Comment:
nit: ordering of logic here isn't consistent with Badge.tsx, can we align them please